js/functional/palindrome.js

```diff
@@ -27,8 +34,20 @@ function isPalindromeUsingLoop(value) {
  * @param {number} index
  */
 function isPalindromeUsingRecursive(value, index = 0) {
-  // write your code here
-  return true;
+  const lowerValue = value.toLowerCase();
+
+  // if the length of current string is less than equal to 1, auto palindrome
```

Comment:
You don't need to write this comment again when the code itself is understood well. Adding more comments like this will make your code seems too complicated to understand.